static/app/views/performance/newTraceDetails/virtualizedViewManager.tsx

```diff
@@ -664,7 +665,7 @@ export class VirtualizedViewManager {
     let max = Number.NEGATIVE_INFINITY;
     let innerMostNode: TraceTreeNode<any> | undefined;
 
-    for (let i = 0; i < this.columns.span_list.column_refs.length; i++) {
+    for (let i = 5; i < this.columns.span_list.column_refs.length - 5; i++) {
```

Comment:
magic number?

Reply:
can be a constant or soemthing?

Reply:
It's magic for now, we store it as the overscroll. It can go a bit under/over, but it's a good approximation. I'm going to try and find a more precise way to measure this in the future